contracts/forwarding/IForwarderV2.sol

```diff
@@ -0,0 +1,13 @@
+/*
+ * SPDX-License-Identifier:    MIT
+ */
+
+pragma solidity ^0.4.24;
+
+import "./IForwarderPayable.sol";
+
+
+// TODO: Cannot inherit interfaces in Solidity 0.4.24
+contract IForwarderV2 is IForwarderPayable {
+    function forward(bytes evmCallScript, bytes context) external payable;
```

Comment:
Do we still need this to be `payable` with the current work on the per-transaction fees?

Reply:
No, but I think it's great to provide all the possible flavors, do you agree?